src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private final Random random = new Random();
+    private final ColorSupplier supplier = new ColorSupplier();
+
+    Ball getRandomBall() {
+        int constant = 100;
+        Ball ball = new Ball(random.nextInt(constant), supplier.getRandomColor());
```

Comment:
![image](https://user-images.githubusercontent.com/117547096/205486250-d7c78b51-36af-4d90-b767-6799a07a7a00.png)

"constant" is not a name)
```suggestion
    private final ColorSupplier supplier = new ColorSupplier();
    private static final int BALL_NUMBER_LIMIT = 100;
    
    Ball getRandomBall() {
        Ball ball = new Ball(random.nextInt(BALL_NUMBER_LIMIT), supplier.getRandomColor());
```